tests/e2e/hcp_external_auth_test.go

```diff
@@ -193,5 +193,55 @@ var _ = Describe("Rosacli Testing", func() {
 					}
 				}
 			})
+
+		It("validation should work when create/list/delete idp and user/admin to external_auth_config cluster via ROSA client - [id:71946]",
+			labels.Medium,
+			func() {
+
+				By("Check if cluster is hosted cp cluster and external_auth_providers is enabled")
+				isHostedCP, err := clusterService.IsHostedCPCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				isExternalAuthEnabled, err := clusterService.IsExternalAuthenticationEnabled(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				if isHostedCP && isExternalAuthEnabled {
```

Comment:
nit: I think it might be a little cleaner if we used the if to check if it's not an HCP cluster with ExternalAuth and skip running the test in that case rather than having the whole thing inside the if block, e.g.:

```
if !isHostedCP || !isExternalAuthEnabled {
    	Skip("This case is for HCP clusters with External Auth")
}
```

Reply:
Thanks for it, Updated likewise